### PR TITLE
Allow extensions to use new API and it get included in Language Packs

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
This leverages the l10n-dev package to analyze ts files for `l10n.t` calls.

### CODE REVIEW

File: build/lib/i18n.ts [hunk 1]
```diff
@@ -608,7 +634,11 @@ export function createXlfFilesForExtensions(): ThroughStream {
 			}
 			return _l10nMap;
 		}
-		gulp.src([`.build/extensions/${extensionName}/package.nls.json`, `.build/extensions/${extensionName}/**/nls.metadata.json`], { allowEmpty: true }).pipe(through(function (file: File) {
+		merge(
+			gulp.src([`.build/extensions/${extensionName}/package.nls.json`, `.build/extensions/${extensionName}/**/nls.metadata.json`], { allowEmpty: true }),
+			createL10nBundleForExtension(extensionName)
+		).pipe(through(function (file: File) {
+			console.log(file.basename);
```
Comment (rzhao271) [inline build/lib/i18n.ts:641]: Intentional?

File: build/lib/i18n.js [hunk 2]
```diff
@@ -530,7 +552,8 @@ function createXlfFilesForExtensions() {
             }
             return _l10nMap;
         }
-        gulp.src([`.build/extensions/${extensionName}/package.nls.json`, `.build/extensions/${extensionName}/**/nls.metadata.json`], { allowEmpty: true }).pipe((0, event_stream_1.through)(function (file) {
+        (0, event_stream_1.merge)(gulp.src([`.build/extensions/${extensionName}/package.nls.json`, `.build/extensions/${extensionName}/**/nls.metadata.json`], { allowEmpty: true }), createL10nBundleForExtension(extensionName)).pipe((0, event_stream_1.through)(function (file) {
+            console.log(file.basename);
```
Comment (TylerLeonhardt) [inline build/lib/i18n.js:556]:
```suggestion
```

File: build/lib/i18n.ts (hunk 1 above)
Comment (TylerLeonhardt) [reply build/lib/i18n.ts:641]:
```suggestion
```

Comment (TylerLeonhardt) [reply build/lib/i18n.ts]: oops :)

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**